File: C/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

void renderLine(int lenght, int spaces);

int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }while(height<1);

    for(int i=0;i<height;i++){
        renderLine(height,height-1-i);
        printf("\n");
    }
}

void renderLine(int length, int spaces)
{
    for (int i = 0; i < length; i++)
    {
        printf("%c", i<spaces ? ' ' : '#');
    }
}

File: Python/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height > 0 and height < 9:
        break

for i in range(height):
    spaces = height-i-1
    for j in range(height):
        if j < spaces:
            print(" ", end="")
        else:
            print("#", end="")
    print()
